pokemon/pom.xml

```diff
@@ -71,5 +71,15 @@
         </plugin>
       </plugins>
     </pluginManagement>
+      <plugins>
+          <plugin>
+              <groupId>org.apache.maven.plugins</groupId>
+              <artifactId>maven-compiler-plugin</artifactId>
+              <configuration>
+                  <source>8</source>
+                  <target>8</target>
+              </configuration>
+          </plugin>

```

Comment:
just for my knowledge what's this for and/or do?

Reply:
i think it's a file created by maven that i accidentally added into the git repo, woops!

Reply:
actually did some googling apparently the pom.xml file tells plugins and dependencys